docs/sphinx/source/changelog/v2.0.5.rst

```diff
@@ -1,6 +1,6 @@
-**************************
-v2.0.5 (December 30, 2020)
-**************************
+**************************************************
+v2.0.5 (2020-12-30) and v2.1.0-beta.2 (2021-01-29)
```

Comment:
the other entries use `%B %d, %Y` but this is `%Y-%m-%d`

Reply:
I think we may have changed that to get it on one line?